<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirProcess</class>
 <widget class="QDialog" name="DirProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DirProcess{
background:#f3f3f3;
}</string>
  </property>
  <widget class="QWidget" name="widget_info" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>430</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget_info{
background:#FBFBFB;
border-radius:5px;
border:1px solid #E5E5E5;
}</string>
   </property>
   <widget class="QLabel" name="label_icon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_icon{
background:#f1f1f1;
}</string>
    </property>
    <property name="text">
     <string>icon</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dir_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_dirname{
border:1px solid #ECECEC;
height:25px;
line-height:25px;
border-bottom:1px solid #8B8B8B;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>目录名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_file_count">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>42</y>
      <width>340</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 个文件夹， 0 个文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_preview" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>430</width>
     <height>191</height>
    </rect>
   </property>
   <widget class="QLabel" name="btn_time_tip">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_time_tip{
background:#ffffff;
color:#000;
}</string>
    </property>
    <property name="text">
     <string>用时: 0 分 0 秒</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget_detail" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>430</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_detail{
background:#fff;
border:1px solid #E5E5E5;
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>9</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>更多详情</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="detail_content" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>38</y>
      <width>430</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#detail_content{
background:#f4f4f4;
border:1px solid #E5E5E5;
border-top:0px;
}</string>
    </property>
    <widget class="QLabel" name="_label_dir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>路径:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dir_path">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>15</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="_label_dir_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>大小:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_size_tip">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0 MB （0字节）</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_loading">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>loading</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
